### PR TITLE
Insteon local

### PR DESCRIPTION
**Pull request by wardcraigj**
**Description:** Add documentation for Insteon Local component

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#5088

### CODE REVIEW

File: source/_components/insteon_local.markdown [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+To integrate your Insteon(local) with Home Assistant, add the following section to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml platform entry
+insteon_local:
+  host: YOUR HUB IP
+  username: YOUR HUB USERNAME
+  password: YOUR HUB PASSWORD
+# Example configuration.yaml light entry
```
Comment (Landrash) [inline source/_components/insteon_local.markdown]: I Suggest adding a line break in-between examples.

File: source/_components/insteon_local.markdown [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+To integrate your Insteon(local) with Home Assistant, add the following section to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml platform entry
+insteon_local:
+  host: YOUR HUB IP
+  username: YOUR HUB USERNAME
+  password: YOUR HUB PASSWORD
+# Example configuration.yaml light entry  
+light:
```
Comment (Landrash) [inline source/_components/insteon_local.markdown]: Light and switch examples should be excluded from this file.
Optionally have a basic example and a full configuration example.

File: source/_components/insteon_local.markdown [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+         device_id: 30DA8A
+         name: Dining Room
+       living_room:
+         device_id: 30D927
+         name: Living Room
+```
+
+Configuration variables:
+
+- **username** (*Required*): The username used to access the Insteon interface (find in your insteon app).
```
Comment (Landrash) [inline source/_components/insteon_local.markdown]: insteon should probably be Insteon.

File: source/_components/insteon_local.markdown [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+title: "Insteon(local)"
+description: "Instructions how to setup the Insteon Hub locally within Home Assistant."
+date:  2016-12-18 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: insteon.png
+ha_category: Hub
+ha_iot_class: "Local Polling"
```
Comment (Landrash) [inline source/_components/insteon_local.markdown]: `ha_version:` tag will need to be added to all new files to reflect the version the component was added in.

Comment (wardcraigj) [inline source/_components/insteon_local.markdown]: Thanks @Landrash! `ha_version` should be add to the other pages as well

**Pull request merged**